test/browser/vitest.config.mts

```diff
@@ -102,8 +103,8 @@ export default defineConfig({
     },
     open: false,
     diff: './custom-diff-config.ts',
-    outputFile: './browser.json',
-    reporters: ['json', {
+    outputFile,
```

Comment:
I don't understand why this needs to be conditional when we support objects if you use several reporters:

```ts
outputFile: {
  html: './html/index.html',
  json: './browser.json'
}
```